core/src/main/java/org/mskcc/cbio/portal/util/DebugHrrsFilter.java

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Can we relocate this class into the portal module? We hope to drop the core module, so adding classes there means that we will need to relocate them later.

Reply:
@sheridancbio do you have a suggestion for where inside the portal module? Looking through the portal module briefly, I don't see a clear subdirectory it belongs in. 

Also I'm using the GlobalPropertiesUtil class from the core module to read in the logging directory portal property.

Reply:
we explored code together .. there are other filters in core that are referenced in the portal web.xml, so this is just a new sibling to those previous dependencies. All will need to be relocated.